### PR TITLE
Correcting the typo for docs ✍️ 

### PR DESCRIPTION
**Pull request by traviswang1997**
**Purpose**: Correcting the typo for docs ✍️ 

_Updates:_
- README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,7 +186,7 @@ export function SmartEdgeWithButtonLabel(props) {
 
 ## Advanced Custom Smart Edges
 
-The `getSmartEdge` function also accepts an optional object `options`, which let's you configure aspects of the path-finding algorithm. You may use it like so:
+The `getSmartEdge` function also accepts an optional object `options`, which allows you configure aspects of the path-finding algorithm. You may use it like so:
```
Comment (tisoap) [inline README.md:189]: typo: Missing a "to": "which allows you **to** configure"

File: README.md [hunk 2]
```diff
@@ -212,10 +212,10 @@ const getSmartEdgeResponse = getSmartEdge({
 
 The `options` object accepts the following keys (they're all optional):
 
-- `nodePadding`: How many pixels of padding is added around nodes, or by how much should the edge avoid the walls of a node. Default `10`, minimum `2`.
-- `gridRatio`: The size in pixels of each square grid cell used for path finding. Smaller values for a more accurate path, bigger for faster path finding. Default `10`, minimum `2`.
-- `drawEdge`: Let's you change the function responsible to draw the SVG line, by default it's the same used by `SmartBezierEdge` ([more bellow](#drawedge))
-- `generatePath`: Let's you change the function that does the path-finding, by default it's the same used by `SmartBezierEdge` ([more bellow](#generatepath))
+- `nodePadding`: How many pixels of padding are added around nodes, or by how much should the edge isolated from the walls of a node. Default `10`, minimum `2`.
```
Comment (tisoap) [inline README.md:215]: issue: "isolated from" doesn't read better than "avoid" here

**Review by tisoap — CHANGES_REQUESTED** [review]
Thanks @traviswang1997 ! I've left a few comments to address before I can merge this

File: README.md (hunk 1 above)
Comment (traviswang1997) [reply README.md]: Thank you @tisoap , I have updated the README now.

**Review by tisoap — APPROVED** [review]

**Pull request merged**